### ISSUE TITLE
Refactor commonalities between MDM.m, RA.m

### ISSUE THREAD

**Issue by shippy** [YaleDecisionNeuro/PsychTaskFramework#41]
There's significant duplication between the two main task scripts. Most of it can be factored out (see e.g. #36).

**Comment by shippy** [comment]
**Candidates for immediate refactor, in decreasing order of feasibility:**
* [ ] "Load or create" / practice bifurcation is the same across tasks
* [ ] Practice run generation / execution (does it really need two forks?)
* [ ] Trial / block generation is tougher (its own issue is #74, but see also #93)

**Comment by shippy** [comment]
After #118, this seems much more doable.
